src/communication/cmconnection.cpp

```diff
@@ -16,6 +16,27 @@
 
 namespace aos::mp::communication {
 
+namespace {
```

Comment:
empty libe at the beginning and end of namespace

Reply:
Done